### DISCUSSION TITLE
Add fixer to split multiple attributes in one opening attribute tag

### DISCUSSION THREAD

**Discussion by mvorisek** [PHP-CS-Fixer/PHP-CS-Fixer#8376] [Feature ideas]
## Feature request

```diff
-#[A, B]
+#[A]
+#[B]
```

**Comment by Wirone** [comment]
I think such a fixer should support both strategies and depending on the config it should group or ungroup attributes.

**Comment by julienfalque** [comment]
[PER](https://www.php-fig.org/per/coding-style/#123-compound-attributes) allows both separate and grouped attributes but:

1. grouped attributes must be on the same line
2. attributes with multiline arguments must not be grouped

I think it would be nice if such fixer would support there rules.

I wonder what should be done for (1) though. Should we force grouped attributes to be on the same line when they can be and separate them otherwise? Always separate?

**Comment by Wirone** [comment]
> I wonder what should be done for (1) though. Should we force grouped attributes to be on the same line when they can be and separate them otherwise? Always separate?

The best would be a configurable strategy (single line vs. multi line). PER ruleset could use the former, while people could reconfigure it to use the latter. For multiline I would use this by default:

```php
#[
    Foo(),
    Bar(),
]
```

but probably we could just enforce new line after comma and let separate fixer do the reformatting 🤔?
